src/content/docs/style-guide/writing-strategies/install-docs-guidance.mdx

```diff
@@ -74,7 +74,7 @@ CodeStream has its own installation method.
 
 [Guided install](/docs/new-relic-one/install-configure/new-relic-guided-install-overview/) provides a single command you can run on your system to install New Relic and start monitoring a variety of things.
 
-At present, guided install installs the infrastructure agent, configures a certain set of integrations, and enables sending logs. It also detects integrations on your system and prompts you to choose whether to monitor those integrations or not.
+At present, guided install installs the infrastructure agent and enables sending logs. It also detects integrations on a system and prompts you to choose whether to monitor those integrations or not. If guided install detects an application of a supported language, it can install the APM agent for that language.
```

Comment:
Might consider rewording to avoid 'install installs'. This isn't a great example but something small like:

At present, when someone opts for the guided install, they get the infrastructure agent with logs yadda yadda